src/any_llm/any_llm.py

```
@@ -203,6 +203,36 @@ def create(
         """
         return cls._create_provider(provider, api_key=api_key, api_base=api_base, **kwargs)
 
+    @classmethod
+    def create_openai_compatible(cls, name: str, api_base: str, api_key: str | None = None, **kwargs: Any) -> AnyLLM:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)